getting_started/scripting/c_sharp/c_sharp_basics.rst

```diff
@@ -161,6 +161,20 @@ located in the project root:
     </Project>
 
 
-Then using the dotnet command line to restore package, in the project root::
+Whenever packages are added or modified, run nuget restore in the root of the
+project directory, to ensure that the nuget packages will be available for msbuild
+(the tool used by godot to build mono projects) to use, run::
 
-    $ dotnet restore
+  $ msbuild /t:restore
+
+
+Debugging Failing Godot Builds
+------------------------------
+
+If you find that your mono project is failing to build, you can debug more closely
+by running the following on the command line, in the root of your project::
+
+  $ msbuild
+
+This prints detailed output on why the build failed. Note that as mentioned
+in the above section, you need to run msbuild /t:restore separately.
```

Comment:
I'm not sure about this. We actually save the msbuild log, so it would be better to give an option to open it.

Reply:
Where is the msbuild log saved? I had trouble finding the documentation on that.

I might also be interested in adding a PR to godot to make that easier to access too. But documentation is a big win for cheap.

Reply:
Sorry for the delayed reply. I added a button to open the build log in: https://github.com/godotengine/godot/commit/4a021fa7a2fdafe4d8aaced8652fa0c72a2a7b08